core/__tests__/models/property/property.ts

```diff
@@ -728,6 +729,53 @@ describe("models/property", () => {
       queryCounter = 0;
     });
 
+    describe("mapped though a non-unique property", () => {
+      let property: Property;
+
+      beforeAll(async () => {
+        property = await helper.factories.property(
+          source,
+          { key: "wordInSpanish" },
+          { column: "spanishWord" }
+        );
+      });
+
+      beforeEach(async () => {
+        await property.update({ unique: false, isArray: false });
+      });
+
+      afterAll(async () => {
+        await property.destroy();
+        await source.setMapping({ id: "userId" });
+      });
+
+      test("properties mapped though unique properties can be unique", async () => {
```

Comment:
very nitpicky, but this small typo is copied multiple times :)
```suggestion
      test("properties mapped through unique properties can be unique", async () => {
```

Reply:
Typo! fixed up in f06019c64299b7e900fa3ae4e7ddfbe49b875798